app/scripts/angular-apimock.js

```diff
@@ -45,13 +45,49 @@ angular.module('apiMock', [])
 		var config = {
 			mockDataPath: '/mock_data',
 			apiPath: '/api',
-			disable: false
+			disable: false,
+			stripQueries: true
 		};
 		var fallbacks = [];
 
 		// Helper methods
 		//
 
+		function serialize(obj) {
+			var str = [];
+
+			obj = sortObjPropertiesAlpha(obj);
+			for(var p in obj){
+				if (obj.hasOwnProperty(p)) {
+					var value = encodeURIComponent(obj[p]);
```

Comment:
It would be good with a test that shows the expected behaviour of using `encodeURIComponent`. I.e. a unit test that shows "what bugs does it protect us from". Like [these](http://stackoverflow.com/questions/1976007/what-characters-are-forbidden-in-windows-and-linux-directory-names) I guess.